README.md

```diff
@@ -5,31 +5,88 @@
+- It will immediately broadcast your peer data via pubsub and repeat the broadcast on the configured `interval`
 
 ### Security Considerations
-It is worth noting that this module does not include any message signing for queries. The reason for this is that libp2p-pubsub supports message signing and enables it by default, which means the message you received has been verified to be from the originator, so we can trust that the peer information we have received is indeed from the peer who owns it. This doesn't mean the peer can't falsify its own records, but this module isn't currently concerned with that scenario.
+It is worth noting that this module does not include any message signing for broadcasts. The reason for this is that libp2p-pubsub supports message signing and enables it by default, which means the message you received has been verified to be from the originator, so we can trust that the peer information we have received is indeed from the peer who owns it. This doesn't mean the peer can't falsify its own records, but this module isn't currently concerned with that scenario.
 
 ## Usage
 
 ### Requirements
 
 This module *MUST* be used on a libp2p node that is running [Pubsub](https://github.com/libp2p/js-libp2p-pubsub). If Pubsub does not exist, or is not running, this module will not work.
 
+### Usage in js-libp2p
+
+See the [js-libp2p configuration docs](https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#customizing-peer-discovery) for how to include this module as a peer discovery module in js-libp2p.
+
+If you are only interested in listening to the global pubsub topic the minimal configuration for using this with libp2p is:
+```js
+const Libp2p = require('libp2p')
+const Websockets = require('libp2p-websockets')
+const MPLEX = require('libp2p-mplex')
+const SECIO = require('libp2p-secio')
+const GossipSub = require('libp2p-gossipsub')
+const PubsubPeerDiscovery = require('libp2p-pubsub-peer-discovery')
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [Websockets], // Any libp2p transport(s) can be used
+    streamMuxer: [MPLEX],
+    connEncryption: [SECIO],
+    pubsub: GossipSub, // Can also be `libp2p-floodsub` if desired
+    peerDiscovery: [PubsubPeerDiscovery]
+  }
+})
+```
+
+### Customizing Pubsub Peer Discovery
+
+There are a few options you can use to customize `Pubsub Peer Discovery`. You can see the detailed [options](#options) below.
+
+```js
+// ... Other imports from above
+const PubsubPeerDiscovery = require('libp2p-pubsub-peer-discovery')
+
+// Custom topics
+const topics = [
+  `myApp.${PubsubPeerDiscovery.TOPIC}`, // It's recommended but not required to extend the global space
+  PubsubPeerDiscovery.TOPIC // Include if you want to participate in the global space
+]
+
+const node = await Libp2p.create({
+  modules: { /* See 'Usage in js-libp2p' for this block */ },
+  config: {
+    peerDiscovery: {
+      [PubsubPeerDiscovery.tag]: {
```

Comment:
The tag likely won't change, so doesn't it make sense to just use a string here? This also makes it easier to understand if one's dealing with an application that reads a config file to setup it's libp2p daemon

Reply:
This is important to be exported and used on `js-libp2p`. Otherwise, it will create even more confusion because the users will not know what that string represents

Reply:
I settled on a middle ground here of using the string for the example and adding a comment about it also being available as a static property.